packages/expo-tracking-transparency/src/TrackingTransparency.ts

```diff
@@ -77,7 +77,11 @@ export async function getTrackingPermissionsAsync(): Promise<PermissionResponse>
  * resolve to `granted`.
  */
 export function isAvailable(): boolean {
-  return Platform.OS === 'ios' && parseInt(Platform.Version.toString(), 10) >= 14;
+  return (
+    Platform.OS === 'ios' &&
+    parseInt(Platform.Version.toString(), 10) >= 14 &&
+    ExpoTrackingTransparency
```

Comment:
`ExpoTrackingTransparency ` returns `undefined` if not available, or populated object if it is